src/taco-utils/tacoPackageLoader.ts

```diff
@@ -322,7 +322,7 @@ module TacoUtility {
         private static createTacoPackageInstallRequest(packageKey: string, dependencyConfigPath: string, logLevel: InstallLogLevel): IPackageInstallRequest {
             if (fs.existsSync(dependencyConfigPath)) {
                 try {
-                    var dependencyLookup: any = require(dependencyConfigPath);
+                    var dependencyLookup: any = JSON.parse(<any> fs.readFileSync(dependencyConfigPath));
```

Comment:
Can you please put your response as a comment, so we know that we can remove the <any> when we get a newer .d.ts file with the proper typings?

After that :+1: